detox/src/configuration/collectCliConfig.js

```diff
@@ -17,6 +17,7 @@ function collectCliConfig({ argv }) {
     configuration: get('configuration'),
     debugSynchronization: get('debug-synchronization'),
     deviceLaunchArgs: get('device-launch-args'),
+    appLaunchArgs: get('DETOX_APP_LAUNCH_ARGS'),
```

Comment:
I'm not entirely sure I follow, here: the expectation is that `DETOX_APP_LAUNCH_ARGS` would be set as an env-var by user? If so, why overrule the option of a cli arg?

Reply:
Also, in any case -- should we not cover this at some integrative/e2e test point?

Reply:
It is covered in `local-cli/test.test.js`, but you are right — I indeed should change it to `--app-launch-args` property **plus** `DETOX_APP_LAUNCH_ARGS` environment variable. 

The point is that I'd like eventually to rewrite all environment variable names to `DETOX_*` to scope ourselves from the other applications running on users' computers and CI. That's why I thought that we should already give some start for this process so that in next `detox@18` all env vars follow that convention.